### DISCUSSION TITLE
Nanoels build

### DISCUSSION THREAD

**Discussion by Hedning003** [kachurovskiy/nanoels#38] [Q&A]
so my naoels is up and running(,big thanks to max for the work you have done)
Have an issue thou..and i cant figure it out.
when running the lathe and engaging the els on the panel whit the on-button,the motor revs up to high for about 0.3-0.5 secounds and then goes down to the correct speed.
Any idea of what this can be? i cant figure it out.
i can hear the engine going to fast and the settles down..i havnt dared trying threadcutting yet,due to this..i am afraid that it will calculate to run at a specifik rpm,but in reality moves faster at the start

**Comment by kachurovskiy** [comment]
If you start ELS when the spindle is already turning, it takes a bit for the stepper to accelerate. Do you have an open or closed loop stepper?

**Reply by Hedning003** [reply]
i bought the stepper suggested in the manual from stepperonline  -  it's not slow accel..it's to much accel and then slows down-takes pherhaps 0.5 secounds after pushing the button - just moving the topslide whit left/right buttons "by hand" ,then it's not an issue

**Reply by kachurovskiy** [reply]
The normal procedure is to enable ELS before starting the spindle - https://github.com/kachurovskiy/nanoels/blob/main/README.md#turning - otherwise stepper must accelerate to above-normal speed first to "catch up" with the spindle.

**Comment by Hedning003** [comment]
today i tested the threadcutting,just to se if this "acceleration" matter'd or not-found a piece of 30mm aluminium and set the els to 1mm,and the acceleration dosnt matter at al-took maybe 6-7 cuts and it was dead on,so now i am happy
